### DISCUSSION TITLE
Ability to specify multiple classes under one attribute

### DISCUSSION THREAD

**Discussion by VladimirMikulic** [tailwindlabs/tailwindcss#9345] [Ideas]
So, instead of:
```css
disabled:border-new-grey-light disabled:text-new-grey-light disable:something-else
```
we do something like this:
```
disabled:(border-new-grey-light text-new-grey-light something-else)
```

Maintainers could of course choose different syntax.

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Reply by VladimirMikulic** [reply]
Appreciate the swift reply @wongjn!
